src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,138 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node<T> head;
+    private Node<T> tail;
+    private int size;
+
+    static class Node<T> {
```

Comment:
```suggestion
    private static class Node<T> {
```
Private ones (methods, classes) should be after public ones.

Reply:
Create "Unlink" method for linked List.